Makefile

```diff
@@ -40,10 +40,28 @@ mypy:
 lint: black flake8 mypy
 
 test-unit:
-	pytest
+	pytest pyteal tests/unit
 
 build-and-test: check-generate-init lint test-unit
 
+# set NUM_PROCS = auto when the following issue has been fixed https://github.com/algorand/pyteal/issues/199
+NUM_PROCS = 1 
+integration-run:
+	pytest -n $(NUM_PROCS) --durations=10 -sv tests/integration
+
+integration-test: integration-run # graviton-demo-run
+
 # Extras:
 coverage:
 	pytest --cov-report html --cov=pyteal
+
```

Comment:
TODO: probably best to carve out the following for a follow-up PR against graviton@🐗